client/site-profiler/components/metrics-section/index.tsx

```diff
@@ -16,20 +16,20 @@ const Container = styled.div`
 
 const NameSpan = styled.span`
 	font-family: 'SF Pro Text';
-	color: var( --studio-gray-40 );
+	color: var( --studio-gray-20 );
 	font-size: 16px;
 	margin-bottom: 8px;
 `;
 
 const Title = styled.div`
-	font-family: 'SF Pro Display';
+	font-family: 'SF Pro Text';
```

Comment:
After checking how other parts of the code behaves, I think the standard behaviour here is to use `font-family: 'SF Pro Text', $sans`, which will use the [following fonts](https://github.com/Automattic/wp-calypso/blob/fe8c158ec21117ee5922623db61de74e229d689b/packages/typography/styles/variables.scss#L10) in that order as the browser finds them in the system. In this case, because it is not finding "SF Pro Text", it is defaulting to a different font.